src/java.base/share/classes/jdk/internal/math/FloatingDecimal.java

```diff
@@ -32,52 +32,44 @@
 /**
  * A class for converting between ASCII and decimal representations of a single
  * or double precision floating point number. Most conversions are provided via
- * static convenience methods, although a <code>BinaryToASCIIConverter</code>
+ * static convenience methods, although a {@link BinaryToASCIIConverter}
  * instance may be obtained and reused.
  */
-public class FloatingDecimal{
+public class FloatingDecimal {
     //
     // Constants of the implementation;
     // most are IEEE-754 related.
-    // (There are more really boring constants at the end.)
     //
-    static final int    EXP_SHIFT = DoubleConsts.SIGNIFICAND_WIDTH - 1;
-    static final long   FRACT_HOB = ( 1L<<EXP_SHIFT ); // assumed High-Order bit
-    static final long   EXP_ONE   = ((long)DoubleConsts.EXP_BIAS)<<EXP_SHIFT; // exponent of 1.0
-    static final int    MAX_SMALL_BIN_EXP = 62;
-    static final int    MIN_SMALL_BIN_EXP = -( 63 / 3 );
-    static final int    MAX_DECIMAL_DIGITS = 15;
-    static final int    MAX_DECIMAL_EXPONENT = 308;
-    static final int    MIN_DECIMAL_EXPONENT = -324;
-    static final int    MAX_NDIGITS = 1100;
-
-    static final int    SINGLE_EXP_SHIFT  =   FloatConsts.SIGNIFICAND_WIDTH - 1;
-    static final int    SINGLE_FRACT_HOB  =   1<<SINGLE_EXP_SHIFT;
-    static final int    SINGLE_MAX_DECIMAL_DIGITS = 7;
-    static final int    SINGLE_MAX_DECIMAL_EXPONENT = 38;
-    static final int    SINGLE_MIN_DECIMAL_EXPONENT = -45;
-    static final int    SINGLE_MAX_NDIGITS = 200;
-
-    static final int    INT_DECIMAL_DIGITS = 9;
+    private static final int    EXP_SHIFT = DoubleConsts.SIGNIFICAND_WIDTH - 1;
+    private static final long   FRACT_HOB = 1L << EXP_SHIFT; // assumed High-Order bit
+    private static final long   EXP_ONE   = (long) DoubleConsts.EXP_BIAS << EXP_SHIFT; // exponent of 1.0
+    private static final int    MAX_SMALL_BIN_EXP = 62;
+    private static final int    MIN_SMALL_BIN_EXP = -63 / 3;
+    private static final int    MAX_DECIMAL_DIGITS = 15;  // max{n : 10^n <= 2^P}
+    private static final int    FLOG_10_MAX_LONG = 18;  // max{i : 10^i ≤ Long.MAX_VALUE}
+
+    private static final int    SINGLE_EXP_SHIFT  = FloatConsts.SIGNIFICAND_WIDTH - 1;
+    private static final int    SINGLE_FRACT_HOB  = 1 << SINGLE_EXP_SHIFT;
+    private static final int    SINGLE_MAX_DECIMAL_DIGITS = 7;
```

Comment:
```suggestion
    private static final int
            EXP_SHIFT          = DoubleConsts.SIGNIFICAND_WIDTH - 1,
            MAX_SMALL_BIN_EXP  = 62,
            MIN_SMALL_BIN_EXP  = -63 / 3,
            MAX_DECIMAL_DIGITS = 15,  // max{n : 10^n <= 2^P}
            FLOG_10_MAX_LONG   = 18,  // max{i : 10^i ≤ Long.MAX_VALUE}
            SINGLE_EXP_SHIFT   = FloatConsts.SIGNIFICAND_WIDTH - 1,
            SINGLE_FRACT_HOB   = 1 << SINGLE_EXP_SHIFT,
            SINGLE_MAX_DECIMAL_DIGITS = 7;
    private static final long
            FRACT_HOB = 1L << EXP_SHIFT, // assumed High-Order bit
            EXP_ONE   = (long) DoubleConsts.EXP_BIAS << EXP_SHIFT; // exponent of 1.0
```
How about this style?